jobs/bump-lockfile.Jenkinsfile

```diff
@@ -132,15 +131,15 @@ lock(resource: "bump-${params.STREAM}") {
                     if (arch == "x86_64") {
                         shwrap("""
                         cosa buildfetch --arch=${arch} --find-build-for-arch \
-                            --url=${BUILDS_BASE_HTTP_URL}/${branch}/builds
+                            --url=s3://${s3_stream_dir}/builds
```

Comment:
now that we're accessing s3 directly we'll need credentials in our environment. We'd need something like: 

https://github.com/coreos/fedora-coreos-pipeline/blob/eb6806cc69472cd52c69bf0aa3e89b765766c998/jobs/kola-aws.Jenkinsfile#L59-L60